<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndFWUpload</class>
 <widget class="QDialog" name="wndFWUpload">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ARIA - FW Upload</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons/aria_rdk.qrc">
    <normaloff>:/icons/ARIA_Logo_2019_solo_simbolo_250dpi.png</normaloff>:/icons/ARIA_Logo_2019_solo_simbolo_250dpi.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Initial address</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QLineEdit" name="leUploaderSelection"/>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>0x</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4" colspan="2">
    <widget class="QPushButton" name="btnBinSelection">
     <property name="text">
      <string>Select Bin</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Uploader</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="btnUpload">
     <property name="text">
      <string>Upload</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Verification flag</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="leStartAddress"/>
   </item>
   <item row="7" column="0" colspan="6">
    <widget class="QTextEdit" name="teOutput">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <widget class="QPushButton" name="btnUploaderSelection">
     <property name="text">
      <string>Select uploader</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Write-operation flag</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Serial Port</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="leFlagWrite"/>
   </item>
   <item row="8" column="4" colspan="2">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="leBinSelection"/>
   </item>
   <item row="8" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4" colspan="2">
    <widget class="QLineEdit" name="leFlagVerify"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="leFlagStart"/>
   </item>
   <item row="5" column="2" colspan="2">
    <widget class="QComboBox" name="cbSerialPort"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Binary file</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Execute flag</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/aria_rdk.qrc"/>
 </resources>
 <connections/>
</ui>
